### PR TITLE
sdk 0.5.15

### PR DESCRIPTION
**Pull request by awojciak**

### CODE REVIEW

File: src/store/sagas/swap.ts [hunk 1]
```diff
@@ -259,7 +259,7 @@ export function* handleSwap(): Generator {
       pair: new Pair(tokenFrom, tokenTo, PAIRS[networkType][poolIndex].feeTier),
       xToY: isXtoY,
       amount: byAmountIn ? amountIn : amountOut,
-      knownPrice: knownPrice,
+      estimatedPriceAfterSwap: knownPrice,
```
Comment (wojciech-cichocki) [inline src/store/sagas/swap.ts:262]: please change `knownPrice` name. It doesn't make sense now.

**Review by wojciech-cichocki — CHANGES_REQUESTED** [review]
Works incorrectly:
Changing slippage tolerance should not affect to swap button

1%:
Screenshot from 2022-02-14 12-40-42

0.01%
Screenshot from 2022-02-14 12-40-50

File: src/store/sagas/swap.ts (hunk 1 above)
Comment (awojciak) [reply src/store/sagas/swap.ts]: done

File: src/store/consts/utils.ts [hunk 2]
```diff
@@ -596,6 +596,7 @@ export const handleSimulate = async (
       resaultWithFee = resault.add(swapSimulateResault.accumulatedFee)
       poolIndex = findPoolIndex(filteredPools[0].address, pools)
       swapSimulateRouterAmount = resault
+      estimatedPrice = swapSimulateResault.priceAfterSwap
```
Comment (matzayonc) [inline src/store/consts/utils.ts:599]: typo

**Pull request merged**